isis/src/apollo/apps/apolloremrx/apolloremrx.cpp

```diff
@@ -0,0 +1,134 @@
+/** This is free and unencumbered software released into the public domain.
+
+The authors of ISIS do not claim copyright on the contents of this file.
+For more details about the LICENSE terms and the AUTHORS, you will
+find files of those names at the top level of this repository. **/
+
+/* SPDX-License-Identifier: CC0-1.0 */
+
+#include "ProcessByTile.h"
+#include "SpecialPixel.h"
+#include "Pvl.h"
+#include "UserInterface.h"
+#include "Brick.h"
+#include "Apollo.h"
+#include <QString>
+#include <cstdlib>
+
+#include "apolloremrx.h"
+
+using namespace std;
+namespace Isis {
+
+  void cpy(Buffer &in, Buffer &out);
+  static int dim;
+  static bool resvalid;
+  static QString action;
+
+  void apolloremrx(UserInterface &ui) {
+      Cube *cube = new Cube(ui.GetFileName("FROM"), "rw");
+      apolloremrx(cube, ui);
```

Comment:
We need to make sure that if the user entered any input attributes that those get used. Before the cube instantiation line add the following lines:
```   
 CubeAttributeInput inAtt = ui.GetInputAttribute("FROM");
    if (inAtt.bands().size() != 0) {
      cube.setVirtualBands(inAtt.bands());
    }
```